artcommon/artcommonlib/rhcos.py

```
@@ -131,25 +131,26 @@ def get_build_id_from_rhcos_pullspec(pullspec) -> str:
     return build_id
 
 
-def get_latest_layered_rhcos_build(container_conf: dict = None, arch: str = None):
+def get_latest_layered_rhcos_build(container_conf: dict = None, arch: str = None, registry_config: str = None):
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)